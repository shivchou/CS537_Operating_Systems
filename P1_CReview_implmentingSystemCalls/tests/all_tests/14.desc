PART 2: Call getcwd after forking.
