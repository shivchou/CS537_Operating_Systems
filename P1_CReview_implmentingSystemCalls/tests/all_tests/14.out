XV6_TEST_OUTPUT Before forking.
XV6_TEST_OUTPUT getcwd in child returned foobar with return value of 0.
XV6_TEST_OUTPUT getcwd in child returned blah with return value of 0.
XV6_TEST_OUTPUT Parent waited for child.
XV6_TEST_OUTPUT getcwd in parent returned foobar with return value of 0.
XV6_TEST_OUTPUT getcwd in parent returned .. with return value of 0.
